spec/rubocop/cop/naming/rescued_exceptions_variable_name_spec.rb

```diff
@@ -511,6 +511,18 @@ def main
         RUBY
       end
     end
+
+    context 'with generic expression' do
```

Comment:
Can you replace "generic expression" with a more concrete explanation?

Reply:
Done.